isb_lib/core.py

```diff
@@ -21,15 +23,16 @@
     "%Y-%m-%d %H:M:%S",  # e.g 2019-12-08 15:54:00
 ]
 DATEPARSER_SETTINGS = {
-    'DATE_ORDER': 'YMD',
-    'PREFER_DAY_OF_MONTH': 'first',
-    'TIMEZONE': 'UTC',
-    'RETURN_AS_TIMEZONE_AWARE': True
+    "DATE_ORDER": "YMD",
+    "PREFER_DAY_OF_MONTH": "first",
+    "TIMEZONE": "UTC",
+    "RETURN_AS_TIMEZONE_AWARE": True,
```

Comment:
It looked like Black hadn't run on the file previously, so there are a few formatting changes.  This is one of them.